pages/posts/hello-2023--hello-json-schema-community.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Hello 2023, Hello JSON Schema Community"
+date: 2023-01-10
+tags:
+  - Update
+  - News
+type: Community
+cover: /img/posts/2023/hello-2023--hello-json-schema-community/jai.webp
+authors:
+  - name: Benjamin Granados
+    photo: /img/avatars/benjagm.png
+    link: https://twitter.com/benjagm
+    byline: Developer Advocate
+---
+
+Since Ben Hutton joined Postman to work full-time on JSON Schema [the past year](https://blog.postman.com/ben-hutton-joins-postman-to-lead-json-schema-strategy/), other top contributors like Jason Desrosiers, Julian Berman, and Greg Dennis have joined, and now all are full-time supporting JSON Schema. So today, I am excited to announce that I am joining as Developer Advocate with a focus on serving the Community, nurturing relationships with other organizations, launching programs to support and grow the Community, and increasing awareness through content, public speaking, and discussions in Twitter, Linkedin, Stackoverflow or Reddit.
+
+## Who is this guy?
+
+My whole career, APIs have been at the core of everything. In the first part of my career, I focused on the engineering aspect of building APIs, playing all the possible team roles. Later, I focused on API Design and Governance, leading API Programs, and most recently, my focus has been Developer Experience and Developer Relations because this human aspect is the most critical for a technology or product to be successful.
+
+Having said that, I see myself as a mix of 1/3 Software Engineering 1/3 API Strategy 1/3 Developer Relations, but I am constantly learning and testing new things, and today I am investing in Product Strategy/Growth. With this diverse background, I hope to be a good complement to the current JSON Schema Team.
+
+## But who is this guy for real?
+
+I am a father of two who loves to spend time with my family and friends. I love mountaineering and team sports, and I think nothing is impossible if you truly believe it. I love this sentence by Nelson Mandela "It always seems impossible until it's done".
+
+## Like to know more about me?
+
+If you like to know more about me, check out [this intro post](https://www.twilio.com/blog/introducing-twilio-developer-evangelist-benjamin-granados) I wrote when I joined Twilio.
+
+## Let's do this!
+
+2023 will be a fantastic year for JSON Schema, and I am excited to be part of this journey with all of you! I can't wait to start connecting with you in the upcoming [working meetings](https://github.com/json-schema-org/community/discussions/35), in [Slack](https://json-schema.org/slack), and [Github](https://github.com/orgs/json-schema-org/discussions).
+
+>* Email: benjamin.granados@postman.com
+>* Twitter: [@benjagm](https://twitter.com/benjagm)
+>* Linkedin: [benjagranados](https://www.linkedin.com/in/benjagranados/)
+>* Github: [benjagm](https://github.com/benjagm)
+
+Cover photo by [Jai Cano](https://www.jaicano.com/redes) on [Instagram](https://www.instagram.com/jaicano).
```

Comment:
Do you know that you/we have a license to use this photo?

I don't see any creative-commons sort of license neither on Jai's IG page nor on the site, and on the site on the contrary I see:

> Quedando expresamente prohibido al usuario la reproducción, transformación, distribución, comunicación pública, puesta a disposición, extracción, reutilización, reenvío o la utilización de cualquier naturaleza, por cualquier medio o procedimiento, de cualquiera de ellos, salvo en los casos en que esté legalmente permitido o sea autorizado por el titular de los correspondientes derechos.

I don't suspect using it as a header image is fair use, though of course I could be mistaken, but do you have some other license to it that covers it or were you just hoping to use it?